src/foam/u2/view/RichChoiceView.js

```diff
@@ -232,8 +232,11 @@ foam.CLASS({
         .addClass(this.myClass())
         .start()
           .addClass(this.myClass('selection-view'))
+          .enableClass('disabled', this.mode$.map((mode) => mode === foam.u2.DisplayMode.DISABLED))
           .on('click', function() {
-            self.isOpen_ = ! self.isOpen_;
+            if ( self.mode === foam.u2.DisplayMode.RW ) {
```

Comment:
It would also be worth at a later date adding support such that if the view transitions to DISABLED state while its currently open, then it automtically gets closed.

Reply:
https://github.com/foam-framework/foam2/pull/1827